GVFS/GVFS.Common/GVFSLock.cs

```diff
@@ -284,9 +288,17 @@ public void RecordReleaseExternalLockRequested()
                 this.lockHeldExternallyTimeMs = this.lockAcquiredTime.ElapsedMilliseconds;
             }
 
-            public void RecordUpdatePlaceholders(long durationMs)
+            public void RecordUpdatePlaceholders(long durationMs, long updateMs, long deleteEmptyMs, long writeAndFlushMs)
             {
-                this.placeholderUpdateTimeMs = durationMs;
+                this.placeholderTotalUpdateTimeMs = durationMs;
+                this.placeholderUpdateTimeMs = updateMs;

```

Comment:
We're changing the definition of this telemetry point. Do you know if we have any reporting looking at this right now?

Reply:
None that I know of.  The reports I know of were looking at modified paths count and William just asked about the placeholder count which noone said they were using.  I will send a email to verify.